src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GaussianPlusConstant.cpp

```diff
@@ -67,6 +68,13 @@ void GaussianPlusConstant<VolumeDim, Fr>::pup(PUP::er& p) {
   p | inverse_width_;
   p | center_;
 }
+
+template <size_t VolumeDim, typename Fr>
+auto GaussianPlusConstant<VolumeDim, Fr>::get_clone() const noexcept
+    -> std::unique_ptr<DampingFunction<VolumeDim, Fr>> {
+  return std::make_unique<GaussianPlusConstant<VolumeDim, Fr>>(
```

Comment:
If you make the class copyable (which it very easily could and should be) then you can do `make_unique<...>(*this)`

Reply:
I thought, like with MathFunctions, we don't want them to be copyable, so that we force users to avoid copying? But if you're cool with it, I'm fine making them copyable

Reply:
OK, I made the class (and the base class) copyable by no longer deleting the copy constructors. Since now I default everything, I just removed my XXXX = default; statements (copy constructors, move constructors, default constructor, default destructor)...to confirm, is that what I should have done?

Reply:
Yea, but I make poor decisions just like everyone else :) When having a `get_clone` function it's much easier if you can copy construct :shrug: